multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -81,3 +80,8 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 
 # Hyperkube image repository to use.
 # hyperkubeImageRepo: quay.io/coreos/hyperkube
+
+# AWS Tags for cloudformation stack (formatted as a list of key-value pairs)
+#stackTags:
```

Comment:
@orkhanM looks good. Can you squash up w/ a commit message along the lines of `kube-aws: cloudformation stack tag functionality`